docs/snippets/vue/portable-stories-jest-compose-stories.ts.mdx

```diff
@@ -0,0 +1,26 @@
+```ts
+// Button.test.ts
+import { test, expect } from '@jest/globals';
+import { render, screen } from '@testing-library/vue';
+import { composeStories } from '@storybook/vue3';
+
+// Import all stories and the component annotations from the stories file
+import * as stories from './Button.stories';
+
+// Every component that is returned maps 1:1 with the stories,
+// but they already contain all annotations from story, meta, and project levels
+const { Primary, Secondary } = composeStories(stories);
+
+test('renders primary button with default args', () => {
+  render(Primary());
+  const buttonElement = screen.getByText('Text coming from args in stories file!');
+  expect(buttonElement).not.toBeNull();
+});
+
+test('renders primary button with overriden props', () => {
+  // You can override props and they will get merged with values from the story's args
+  render(Primary({ label: 'Hello world' }));
```

Comment:
If https://github.com/storybookjs/storybook/pull/26198 lands as-is, the API for rendering Vue components is changed slightly:

```suggestion
  render(Primary, { props: { label: 'Hello world' } });
```

See [`03ca102` (#26198)](https://github.com/storybookjs/storybook/pull/26198/commits/03ca10229e0b786215aaded00ba7577e6265aa9e)